nextflow.config

```diff
@@ -67,18 +68,19 @@ params {
     fusioncatcher_fusions         = null
 
     // Boilerplate options
-    outdir                     = null
-    tracedir                   = "${params.outdir}/pipeline_info"
-    publish_dir_mode           = 'copy'
-    email                      = null
-    email_on_fail              = null
-    plaintext_email            = false
-    monochrome_logs            = false
-    help                       = false
-    validate_params            = true
-    show_hidden_params         = false
-    schema_ignore_params       = 'genomes'
-    enable_conda               = false
+    outdir                            = null
+    tracedir                          = "${params.outdir}/pipeline_info"
+    publish_dir_mode                  = 'copy'
```

Comment:
should we set this to `move` instead? occupies less space

Reply:
I would maybe let it like this until 2.0.1 because I am sure there will be some debugging involved and I have the feeling `copy `would help with that.